src/controllers/dashboard/plugins/installed/index.js

```diff
@@ -59,7 +79,7 @@ function getPluginCardHtml(plugin, pluginConfigurationPages) {
 
     html += "<div class='cardText'>";
     html += configPage && configPage.DisplayName ? configPage.DisplayName : plugin.Name;
-    html += '</div>';
+    html += '<br/>' + globalize.translate('LabelStatus') + ' ' + plugin.Status + '</div>';
```

Comment:
Same here

Reply:
didn't know if ${globalize.transtion(...) would work so started from the trailing space.